docs/usage/fields.md

```diff
@@ -246,4 +274,99 @@ print(user)
 #> first_name='John' last_name='Doe'
 ```
 
+## Number Constraints
+
+There are fields that can be used to constrain numbers:
+
+* `gt` - greater than
+* `lt` - less than
+* `ge` - greater than or equal to
+* `le` - less than or equal to
+* `multiple_of` - a multiple of the given number
+* `allow_inf_nan` - allow `'inf'`, `'-inf'`, `'nan'` values
+
+Here's an example:
+
+```py
+from pydantic import BaseModel, Field
+class Foo(BaseModel):
+    positive: int = Field(..., gt=0)
+    non_negative: int = Field(..., ge=0)
+    negative: int = Field(..., lt=0)
+    non_positive: int = Field(..., le=0)
+    even: int = Field(..., multiple_of=2)
+    love_for_pydantic: float = Field(..., allow_inf_nan=True)
+foo = Foo(
+    positive=1,
+    non_negative=0,
+    negative=-1,
+    non_positive=0,
+    even=2,
+    love_for_pydantic=float("inf"),
+)
+print(foo)
+#> positive=1 non_negative=0 negative=-1 non_positive=0 even=2 love_for_pydantic=inf
+```
+
+??? info "JSON Schema"
+    In the generated JSON schema:
+
+    - `gt` and `lt` constraints will be translated to `exclusiveMinimum` and `exclusiveMaximum`.
+    - `ge` and `le` constraints will be translated to `minimum` and `maximum`.
+    - `multiple_of` constraint will be translated to `multipleOf`.
+
+    The above snippet will generate the following JSON Schema:
+
+    ```json
+    {
+      "title": "Foo",
+      "type": "object",
+      "properties": {
+        "positive": {
+          "title": "Positive",
+          "type": "integer",
+          "exclusiveMinimum": 0
+        },
+        "non_negative": {
+          "title": "Non Negative",
+          "type": "integer",
+          "minimum": 0
+        },
+        "negative": {
+          "title": "Negative",
+          "type": "integer",
+          "exclusiveMaximum": 0
+        },
+        "non_positive": {
+          "title": "Non Positive",
+          "type": "integer",
+          "maximum": 0
+        },
+        "even": {
+          "title": "Even",
+          "type": "integer",
+          "multipleOf": 2
+        },
+        "love_for_pydantic": {
+          "title": "Love For Pydantic",
+          "type": "number"
+        }
+      },
+      "required": [
+        "positive",
+        "non_negative",
+        "negative",
+        "non_positive",
+        "even",
+        "love_for_pydantic"
+      ]
+    }
+    ```
+
+    Check the [JSON Schema 2020-12] for more details.
```

Comment:
What's going on here — is this supposed to be a link?
```suggestion
    See the [JSON Schema Draft 2020-12 Spec](https://json-schema.org/specification-links.html#2020-12) for more details.
```

Reply:
The link is in the bottom of the markdown.